src/test/java/com/thealgorithms/recursion/FibonacciSeriesTest.java

```diff
@@ -23,5 +24,9 @@ public void testFibonacci() {
         assertEquals(144, FibonacciSeries.fibonacci(12));
         assertEquals(233, FibonacciSeries.fibonacci(13));
         assertEquals(377, FibonacciSeries.fibonacci(14));
+        assertEquals(610, FibonacciSeries.fibonacci(15));
+        assertEquals(6765, FibonacciSeries.fibonacci(20));
+        assertEquals(75025, FibonacciSeries.fibonacci(25));
+        assertEquals(832040, FibonacciSeries.fibonacci(30));
```

Comment:
I think we reached the point when one should express these are parametrized tests. Perhaps even for all Fibonacci sequence implementations. An example to look at is [`NumberOfDigitsTest.java`](https://github.com/TheAlgorithms/Java/blob/1e6ed97fcf96d15556466e371096b0159ffe63af/src/test/java/com/thealgorithms/maths/NumberOfDigitsTest.java).